charts/trino/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.12.0
+version: 0.13.0
```

Comment:
Remove the version update since it cause an automatic release

Reply:
Done in https://github.com/trinodb/charts/pull/98/commits/de2465df0d928cd2580101878c1657be359a1807